chemprop/utils.py

```diff
@@ -707,6 +707,12 @@ def update_prediction_args(
                 setattr(predict_args, key, override_defaults.get(key, value))
 
     # Same number of molecules must be used in training as in making predictions
+    if train_args.number_of_molecules != predict_args.number_of_molecules \
+            and not (isinstance(predict_args, FingerprintArgs) and predict_args.fingerprint_type == "MPN" and predict_args.mpn_shared and predict_args.number_of_molecules == 1):
```

Comment:
fix style with `(...)` over multiple lines rather than `\` for multi-line statements